modules/openapi-generator/src/main/resources/csharp/ApiClient.mustache

```diff
@@ -485,9 +485,8 @@ namespace {{packageName}}.Client
             {
                 var policy = RetryConfiguration.RetryPolicy;
                 var policyResult = policy.ExecuteAndCapture(() => client.Execute(req));
-                response = (policyResult.Outcome == OutcomeType.Successful) ? client.Deserialize<T>(policyResult.Result) : new RestResponse<T>
+                response = (policyResult.Outcome == OutcomeType.Successful) ? client.Deserialize<T>(policyResult.Result) : new RestResponse<T>(req)
```

Comment:
Generally we don't shorten words. What is req? It should likely be renamed to restRequest. I know it's not exactly the overall point of your pr, but these are things that should get fixed.

Reply:
Looks like this has been the case since the original refactor ~5 years ago.
Maybe just `request` instead, as we already have a plain `response` variable in the same method?